src/main/java/Model/BaseballNumbers.java

```diff
@@ -0,0 +1,62 @@
+package Model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BaseballNumbers {
+    private List<Integer> numbers = new ArrayList<>();
+    private static final int maxLength = 3;
+    private static final String overMaxLengthMessage = "숫자 개수 한계치 초과";
```

Comment:
![image](https://user-images.githubusercontent.com/49794401/227779384-fb5e0495-1020-476a-a015-c878b00ad4ad.png)

[공통 피드백 문서](https://docs.google.com/document/d/17V6DEI0ZIivfLYhWz5mt8bL4H4xCQoG-xb4XPYxLiLo/edit)의 `구현 순서도 컨벤션이다`를 참고해볼까요?

```suggestion
    private static final String overMaxLengthMessage = "숫자 개수 한계치 초과";
    private static final int maxLength = 3;
    
    private List<Integer> numbers = new ArrayList<>();
```